DependencyInjection/Configuration.php

```diff
@@ -735,7 +741,7 @@ private function getDeprecationMsg(string $message): array
         if (method_exists(BaseNode::class, 'getDeprecation')) {
             return [
                 'doctrine/doctrine-bundle',
-                '2.0',
+                '2.2',
```

Comment:
This change is wrong. If different methods need different versions, you need to pass that as an argument, not changing the version for previous deprecations

Reply:
it's already wrong in master, will fix in separate commit